src/Message/Transcoder.php

```diff
@@ -41,6 +41,7 @@ final class Transcoder
         '866'                       => 'IBM866',
         'ansi-1251'                 => 'windows-1251',
         'ansi_x3.4-1968'            => 'us-ascii',
+        'ansi_x3.110-1983'          => 'ISO-8859-1',
```

Comment:
According to:

1. https://www.compart.com/en/unicode/charsets/ANSI_X3.110-1983
1. https://stackoverflow.com/questions/16044154/what-should-i-do-with-emails-using-charset-ansi-x3-110-1983

I don't think `ISO-8859-1` should be used as a replacement. I think it would be better to let the user decide this without altering this very library, but of course a development must be done for the goal